components/chart/table-chart.tsx

```diff
@@ -1,4 +1,4 @@
-import { ChartData } from '@/types/chart';
+import { ChartData } from '@/types/chat';
```

Comment:
`chat` and `chart ` are not same. and there will be more types about chart later .